index.html

```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
       </section>
       <section class="contact">
         <p class="contact-title desktop">Let's chat!</p>
-        <p class="contact-title mobile">CONTACT US</p>
+        <p class="contact-title mobile">CONTACT ME</p>
         <p class="contact-description">
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
```

Comment:
- [ ] Please use the correct font-weight for the heading and align the paragraph text in the center and make sure it matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design.

### Your design 

![image](https://user-images.githubusercontent.com/83803414/166551755-a8a64dd9-40ed-4c0d-88c6-dbb9fa6dfe5d.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166551796-5642e96b-2e5f-4335-903d-abd90829204c.png)